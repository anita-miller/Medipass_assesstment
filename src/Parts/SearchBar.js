import React from 'react'
import TextField from '@material-ui/core/TextField';
import Button from '@material-ui/core/Button';

const SearchBar = (props)=>{
    return(
        <div>
            <TextField style={{padding: 20}}
                        id="searchInput"
                        placeholder="Find your pie"
                        margin="normal"
                        onChange={props.onSearchChange}
            />  
        </div>
    )
}


export default SearchBar