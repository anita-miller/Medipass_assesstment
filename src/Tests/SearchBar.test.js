import React from 'react';
import SearchBar from '../Parts/SearchBar';
import renderer from 'react-test-renderer';


test('SearchBar component renders a snapshot', () =>{
    const order = "asc"
    const component = renderer.create(
        <SearchBar 
            order={order}
        />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();



});