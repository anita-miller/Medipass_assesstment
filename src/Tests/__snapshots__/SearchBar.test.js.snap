// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component renders a snapshot 1`] = `
<div>
  <div
    className="MuiFormControl-root-1 MuiFormControl-marginNormal-2"
    style={
      Object {
        "padding": 20,
      }
    }
  >
    <div
      className="MuiInputBase-root-18 MuiInput-root-5 MuiInput-underline-9 MuiInputBase-formControl-19 MuiInput-formControl-6"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        className="MuiInputBase-input-28 MuiInput-input-13"
        disabled={false}
        id="searchInput"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Find your pie"
        required={false}
        type="text"
      />
    </div>
  </div>
</div>
`;
